packages/http-client-csharp/generator/TestProjects/Spector/http/payload/multipart/tspCodeModel.json

```diff
@@ -1787,8 +1787,7 @@
      "readOnly": false,
      "decorators": [],
      "crossLanguageDefinitionId": "Payload.MultiPart.FormData.HttpParts.NonString.float.Request.temperature.anonymous.contentType",
-     "discriminator": false,
```

Comment:
What happened to these properties?

Reply:
they are not model properties, but parameters. Since it is a content type (from its crosslanguagedefnitionid), it should be a header parameter, and a header parameter could never be a discriminator therefore it does not have that property any more